src/utils.h

```diff
@@ -25,6 +25,12 @@
 #define SEND_CREDS_NOTSK 1
 #define SEND_CREDS_FAIL 2
 
+#define SYS_FS_KERNEL_YAMA_PTRACE_SCOPE "/sys/kernel/yama/ptrace_scope"
+#define YAMA_PTRACE_SCOPE_CLASSIC 0
```

Comment:
Let's align these mode names with what we have in Linux kernel https://github.com/torvalds/linux/blob/98369dccd2f8e16bf4c6621053af7aa4821dcf8e/security/yama/yama_lsm.c#L23